controller/licenseController.go

```
@@ -0,0 +1,257 @@
+//	@Produce		json
+//	@Success		201	{object}	dto.DriverLicenseDto
+//	@Failure		400
+//	@Failure		404
+//	@Failure		500
+//	@Param			model	body	dto.DriverLicenseDto	true	"License model"
+//	@Router			/license [post]
+func (c *LicenseController) createLicense(ctx *gin.Context) {
+	var licenseDto dto.DriverLicenseDto
+	if err := ctx.Bind(&licenseDto); err != nil {
+		c.logger.Errorf("Failed to bind error = %+v", err)
+		ctx.AbortWithError(http.StatusBadRequest, err)
```

golangci-lint (1.64.8)

Error return value of `ctx.AbortWithError` is not checked

(errcheck)